virttest/utils_test/libvirt.py

```diff
@@ -855,9 +855,11 @@ def cleanup_pool(self, pool_name, pool_type, pool_target, emulated_image,
                 pool_target = os.path.join(self.tmpdir, pool_target)
                 if os.path.exists(pool_target):
                     shutil.rmtree(pool_target)
+            manage_gluster = kwargs.get("remote_gluster_manage", "yes") == "yes"
             if pool_type == "gluster" or source_format == 'glusterfs':
-                gluster.setup_or_cleanup_gluster(False, source_name,
-                                                 pool_name=pool_name, **kwargs)
+                if manage_gluster:
```

Comment:
Don't need `manage_gluster` variable since you only use it in this conditional. So...
`if kwargs.get("remote_gluster_manage", "yes") == "yes"`
... is enough.

Reply:
@wainersm Thanks. Fixed.